data/net/minecraft/world/item/enchantment/Enchantment.mapping

```diff
@@ -55,7 +55,7 @@ CLASS net/minecraft/world/item/enchantment/Enchantment
 		COMMENT Gets the description Id of the enchantment. This is commonly used to create localization keys.
 	METHOD getFullname (I)Lnet/minecraft/network/chat/Component;
 		COMMENT Gets the name of the enchantment with the level appended to the end. The game will attempt to convert the letter to roman numerals.
-		COMMENT @return The name of the enchantment with the level appened to the end.
+		COMMENT @return The name of the enchantment with the level append to the end.
```

Comment:
I think the typo to be corrected here is `appened` to `appended`, with a missing `d`.
```suggestion
		COMMENT @return The name of the enchantment with the level appended to the end.
```